Rate Limiting info added (#1533)
saucelabs/sauce-docs: docs/mobile-apps/app-storage.md

     * [iOS Demo App](https://github.com/saucelabs/my-demo-app-ios/releases)
     * [Android Demo App](https://github.com/saucelabs/my-demo-app-android/releases)

## Rate Limiting

> To increase service stability and prevent overload by a high volume of incoming traffic, we have set the following rate limits for uploading your mobile apps: 

Author: ```suggestion
To increase service stability and prevent overload by a high volume of incoming traffic, we have set the following rate limits for uploading your mobile apps (effective from 14.11.2022): 
```

     * [iOS Demo App](https://github.com/saucelabs/my-demo-app-ios/releases)
     * [Android Demo App](https://github.com/saucelabs/my-demo-app-android/releases)

## Rate Limiting

> To prevent service overload and outbursts of incoming traffic, Sauce Labs introduced the Rate Limiting feature. It helps maximize the stability of incoming traffic.

Reviewer: ```suggestion
To increase service stability and prevent overload by a high volume of incoming traffic, we have set the following rate limits for uploading your mobile apps: 
```

Reviewer 2: To prevent service overload and maximize our service quality, Sauce Labs will introduce new rate limits effective from 14.11.2022.

> See the table below to learn more about applied rate limits:

Reviewer: ```suggestion

```
